packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -312,12 +312,19 @@ exports.mapPolylangTranslations = entities =>
   entities.map(entity => {
     if (entity.polylang_translations) {
       entity.polylang_translations___NODE = entity.polylang_translations.map(
-        translation =>
-          entities.find(
+        translation => {
+          const post = entities.find(
             t =>
               t.wordpress_id === translation.wordpress_id &&
               entity.__type === t.__type
-          ).id
+          )
+
+          if (!post) {
+            return null
+          }
+
+          return post.id
+        }
       )
```

Comment:
This looks reasonable! 

One question - should we also remove `null`s from `entity.polylang_translations___NODE` array? I'm not sure what happens when we have nulls in that array - do we get nulls in query results then? If so, it would be good to remove those.

Reply:
I'm not sure - I think it returns an array of nulls but I can't confirm it. Removing those might break some other stuff, and it's a little hard to test. For example, I have no clue why this happens at all.. I haven't checked every API response to `/wp-json/wp/v2/posts` (we have 500+ posts), but from the first few pages, they all seem populated correctly.

Reply:
Actually, scratch that. 

It's because it can't find the corresponding post when doing `entities.find`.  It might be because the corresponding ID does not exist anymore or something, and polylang doesn't update it?

Reply:
Post could be in Trash, or set to private, or entirely deleted - in all those cases we wouldn't be able to find the post.

I think the change make sense (just to not crash gatsby in case we can't find post), just want to check how Gatsby currently handles `null`s in `___NODE` arrays (I'm not sure). To me it makes sense to not return those nulls in query results, because its easier to write react code (not to have to check for nulls there)

Reply:
Ok, we are ignoring `null`s in `___NODE` arrays - so this is good to go IMO